econml/inference/_inference.py

```diff
@@ -559,7 +559,7 @@ def __init__(self, d_t, d_y, pred, inf_type, fname_transformer=None,
         # For effect summaries, d_t is None, but the result arrays behave as if d_t=1
         self._d_t = d_t or 1
         self.d_y = d_y
-        self.pred = pred
+        self.pred = np.copy(pred) if pred is not None and not np.isscalar(pred) else pred
```

Comment:
I can't remember if it's this copy or a different one that is necessary, but please add a test that verifies that the problematic code that modifies the estimator's coef_ (or whatever it was) has been fixed.

Reply:
done